### ISSUE TITLE
Ducks? 🦆

### ISSUE THREAD

**Issue by nelsonic** [dwyl/phase-two#109]
Opening this issue as a placeholder for the discussion. Definitely not a _current_ priority. ⏳ 

This is a good intro to _Why_...?
Regenerative Farm Combines Ducks and Blueberries: https://youtu.be/WfqPlDzWwjM **`T12m`**

[![image](https://user-images.githubusercontent.com/194400/162997052-c5eb7584-6885-4026-ad92-4385515415d9.png)](https://youtu.be/WfqPlDzWwjM)

Slug slayers. Egg layers. Natural fertiliser factory. Perfect for organic re-gen farming. 🥚  👩‍🌾 

@iteles 'member Vergenoegd Wine Estate in **Stellenbosch** we visited last time in SA? 🇿🇦 

[![image](https://user-images.githubusercontent.com/194400/162999459-c4f15abf-165b-46b1-937e-3ef064ee6567.png)](https://youtu.be/nRx1yeIQgAM)

> They have a commercial operation with over 1k ducks. It's a major tourist attraction. 📸 

Wouldn't it be cool to test **_one_ field** in **Landim** and transition to Organic from Pesticide-intensive? 💡 
Wonder how much investment would be required for an MVP ...  🤔 
Curious to know how much is spent on pesticides & fertilisers each year ... 💸 
Maybe we could ask your uncle (blueberry farmer) if he has considered having them. 🫐 
The setup in this video above looks very profesh. 😎  Maybe we could reach-out to them for advice.

https://www.parccarreg.com/
![image](https://user-images.githubusercontent.com/194400/163002428-d00e1546-ba2c-40ab-8c0f-ba2b12e07ba8.png)

https://parc-carreg.mailchimpsites.com/
![image](https://user-images.githubusercontent.com/194400/163002530-24139681-4583-4d22-9631-a4d95ab9da96.png)

https://www.parccarreg.com/hatching-eggs
![image](https://user-images.githubusercontent.com/194400/163002596-a9cc05ee-d3e6-4102-a51f-bcf47c564339.png)

10 Reasons Why NOT to Keep Ducks? https://youtu.be/FF8ICU5cskY 👀 
[![image](https://user-images.githubusercontent.com/194400/163002821-3410758d-c29d-455d-a66e-b737f2760e16.png)](https://youtu.be/FF8ICU5cskY)

In this video Mark mostly just complains about the **noise**, mess/smell and ducks not putting themselves "to bed" ... but look at how he is keeping them. Their "pen" is too small and basically mud. He could easily get around the problem by (a) giving them a bigger area and (b) covering it in straw which would make it much easier to collect the free fertiliser! He complains about them eating more (and wasting food) than chickens. I think the idea of having them on a permaculture farm / vineyard is still totally a win.

The comments are definitely worth reading for context:

![image](https://user-images.githubusercontent.com/194400/163004027-907b88e7-e0c5-478b-834f-3b01ad906bcd.png)

In the comments someone proposes Muscovies: https://en.wikipedia.org/wiki/Muscovy_duck
![image](https://user-images.githubusercontent.com/194400/163003049-f9cb1f5c-ac33-460f-91a8-22811873beac.png)

![image](https://user-images.githubusercontent.com/194400/163003592-12dfc38e-8d55-44c1-8202-b01d341b304b.png)

The fact that Moscovy ducks are quiet is a definite selling point for me. 🙉 
But then I don't think we would have to deal with the noise on a daily basis ... 💭 

But there are probably some EU regs on what type of Ducks are _allowed_ for avian flu control, etc.

What I wish I’d known before starting a Duck Farm: https://youtu.be/mYmBQ-gv9gE **`T13m`**
[![image](https://user-images.githubusercontent.com/194400/163006408-7687f062-cf16-4b24-942c-c112e6e2368d.png)](https://youtu.be/mYmBQ-gv9gE)
Doing this in northern USA (Vermont) looks expensive (unprofitable) ... heating the barn etc.
Bet they make more revenue from YT (500k+ subs) than they do from their farm. 💰 

Speaking of revenue, "How to make $2,000 a month with 25 ducks!": https://youtu.be/QzLHoLu5VyY?t=286
[![image](https://user-images.githubusercontent.com/194400/163006945-596634bb-6aed-4080-b8ef-73063603e883.png)](https://youtu.be/QzLHoLu5VyY?t=286)
This one is very slow. Increase the speed to **1.5** if you watch it.
But basically he sells the hatchlings for more than he would get for the eggs. 
With 25 [**Khaki Campbell** ducks](https://en.wikipedia.org/wiki/Khaki_Campbell) he was getting 30+ eggs a day. 
He would sell them for $20 for 10 and ship 3 boxes of eggs  per day.
So **$1800/month** not quite **$2k** but not far off.
So that would require an incubation/hatching hut. **~€1k**

Why Every [Organic] Vegetable Garden Needs Ducks: https://youtu.be/baXSrl5Vmb0
[![image](https://user-images.githubusercontent.com/194400/163010185-80321c44-8b4f-4edf-88bc-2cf7f856a66c.png)](https://youtu.be/baXSrl5Vmb0)
Basically pest control with free fertiliser. 

Gotta love the YT comments:
![image](https://user-images.githubusercontent.com/194400/163010450-dabd9bd6-4842-446e-9de1-9a1c9688c043.png)

My initial conclusion is buying ducklings is the way to go. 🐥 
Perhaps we could also speak to **Pete** (Gerês) about it ... ☀️  
But I wouldn't keep them as pets the way **Pete** does ... rather I'd have them as slug-slayers.
Though they are mega-cute as chicks:

![image](https://user-images.githubusercontent.com/194400/163012196-e4b4e5b0-9daa-4ba6-8b74-e65e742caf62.png)

Keen to hear/read your thoughts at some point ... Again, not urgent. 👍

**Comment by iteles** [comment]
This is a very cool idea! I'm keen, especially on a property with a body of water because I'd feel bad keeping water birds on land.

I'd start by speaking to Pete about it 👍 

Do you think the dogs in Landim would attack them? 😬 🙃

**Comment by nelsonic** [comment]
@iteles yeah, the Landim (Rescue) Dogs would 100% obliterate the ducks given half a chance! Well remembered. 🐕 
They weren't raised/trained with Ducks so they will see them as a snack instead of as team members. 🤦‍♂️ 
However, Ducks have one thing going for them that chickens don't, they fly!! So they can escape. 💭 

Definitely speak to Pete next time we're in Gerês. 

As for the body of **water**. #21  While I **_totally_ agree** with having water fowl in their natural habitat and would love to have them on _our_ land which will 100% border a lake/river ... having them on a **vineyard** to help with pest control so that you can have an **Organic** ecosystem would mean that you _don't_ want to give them "too much" water otherwise they would just spend all day "frolicking" in the water and not do their "job". 
Rather you want a couple of man-made (easy to clean) rain water fed **ponds** that the ducks have access to near to their sleeping quarters. There would need to be a cover over the pond to _exclude_ the ducks from it for a couple of hours a day to encourage them to go foraging.

**Comment by nelsonic** [comment]
Duck Rice Farming in Japan: https://youtu.be/SNR_3GeUoqI

**Comment by nelsonic** [comment]
<img width="590" alt="image" src="https://user-images.githubusercontent.com/194400/188431703-96aaaf96-4010-477e-a3bd-af1e28576a7b.png">

Would land bordering a major year-round river be suitable? 😜 #116

**Comment by nelsonic** [comment]
MAKE A CITY WAGE FROM A SMALL FARM S5 ● E42: https://youtu.be/4rEJ0uGfpJM?t=970
[![image](https://user-images.githubusercontent.com/194400/194784756-b991ce20-83cf-4fc2-81c0-7955b1249cbc.png)](https://youtu.be/4rEJ0uGfpJM)

**Comment by nelsonic** [comment]
https://github.com/ESP32-COOP/ESP32-COOP-DOC
via: 
https://news.ycombinator.com/item?id=37320037

**Comment by nelsonic** [comment]
Vital Farms: https://youtu.be/Cr0VagmhhNk
